src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaBuilderTests.cs

```diff
@@ -242,74 +236,77 @@ public void BuildDeltaWithContractEntries()
             var expectedBytesToHash = BuildExpectedBytesToHash(selectedTransactions, shuffledEntriesBytes);
 
             var deltaBuilder = new DeltaBuilder(transactionRetriever, _randomFactory, _hashProvider, _peerSettings,
-                _cache, _dateTimeProvider, _logger);
+                _cache, _dateTimeProvider, _mapperProvider, _logger);
             var candidate = deltaBuilder.BuildCandidateDelta(_previousDeltaHash);
 
             ValidateDeltaCandidate(candidate, expectedBytesToHash);
 
             _cache.Received(1).AddLocalDelta(Arg.Is(candidate), Arg.Any<Delta>());
         }
 
-        [Fact]
-        public void When_contract_entries_exceed_delta_gas_limit_some_entries_are_ignored()
+        //[Fact]
+        //public void When_contract_entries_exceed_delta_gas_limit_some_entries_are_ignored()
+        //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)